role-service/src/main/resources/db/changelog/changelog-permission-data.xml

```diff
@@ -17,6 +17,13 @@
             <column name="modified_by" value="katesan"/>
         </insert>
 
+        <insert tableName="permission">
+            <column name="id" value="link-user"/>
+            <column name="created_by" value="katesan"/>
```

Comment:
I think it is better to use your own alias...

Reply:
ok will do..